libraries/DateTime/test/dateTimeLibrary.js

```diff
@@ -0,0 +1,359 @@
+//@ts-nocheck
+const fs = require('fs-extra')
+const path = require('path')
+const Deployer = require('forgae-lib').Deployer
+const contractDirectory = `${process.cwd()}/contracts/`;
+const exampleContract = 'ExampleContract.aes'
```

Comment:
why test against the example wrapper?

Reply:
I'd like to emphasize on how namespaces could be used from a separate contract